### ISSUE TITLE
sendAnonymousUsage has inverse meaning in v2.0.0-alpha3

### ISSUE THREAD

**Issue by powerman** [containous/traefik#4740]
### Do you want to request a *feature* or report a *bug*?

Bug

### Did you try using a 1.7.x configuration for the version 2.0?

- [ ] Yes
- [x] No

### What did you do?

`--global.sendAnonymousUsage=false`

### What did you expect to see?

```
level=info msg="\nStats collection is disabled.\nHelp us improve Traefik by turning this feature on :)\nMore details on: https://docs.traefik.io/basics/#collected-data\n"
```

### What did you see instead?

```
level=error msg="\nYou haven't specify the sendAnonymousUsage option, it will be enable by default.\n"
level=info msg="\nStats collection is enabled.\nMany thanks for contributing to Traefik's improvement by allowing us to receive anonymous information from your configuration.\nHelp us improve Traefik by leaving this feature on :)\nMore details on: https://docs.traefik.io/basics/#collected-data\n"
```

### Output of `traefik version`: (_What version of Traefik are you using?_)

```
Version:      2.0.0-alpha3
Codename:     faisselle
Go version:   go1.12.1
Built:        2019-03-29T17:53:25Z
OS/Arch:      linux/amd64
```

### What is your environment & configuration (arguments, toml, provider, platform, ...)?

No config file, only arguments:
```yaml
    traefik:
        image: traefik:2.0
        command: >
            --api
            --entrypoints='Name:web Address::8000'
            --metrics
            --metrics.prometheus
            --providers
            --providers.docker
            --providers.docker.exposedbydefault=false
            --providers.docker.watch
            --global.sendAnonymousUsage=false
            --global.debug
            --log.loglevel=debug
            --accesslog
```

### Workaround:

With `--global.sendAnonymousUsage=true` I get in logs:
```
level=info msg="\nStats collection is disabled.\nHelp us improve Traefik by turning this feature on :)\nMore details on: https://docs.traefik.io/basics/#collected-data\n"
```
But I'm not sure is actual stats enabled or disabled - something is inversed, but is it only log message or both log message and sending stats?

**Comment by jbdoumenjou** [comment]
Hi @powerman, thank you for pointing out this issue.
This behavior is due to the way the command line is parsed.
We will fix this behavior as soon as possible.
Until this fix, a workaround is to use the toml file configuration which works as expected.

**Comment by traefiker** [comment]
Closed by #4935.
